cfgov/unprocessed/css/molecules/related-metadata.less

```diff
@@ -0,0 +1,83 @@
+/* topdoc
+  name: Related Metadata
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+    <div class="m-related-metadata">
+        <h2 class="header-slug">
+            <span class="header-slug_inner">
+                Related Metadata
+            </span>
+        </h2>
+        <div class="m-related-metadata_item-container">
+            <h4></h4>
+            <ul class="list
+                       list__unstyled
+                       list__links">
+                <li class="list_item">
+                    <a class="list_link">
+                    </a>
+                </li>
+            </ul>
+            <p class="date u-mb0"></p>
+            <p class="short-desc u-mb0"></p>
+        </div>
+    </div>
+    codenotes:
+      - |
+        Structural cheat sheet:
+        -----------------------
+        .m-related-metadata
+          .header-slug
+            .header-slug_inner
+        .m-related-metadata_item-container
+          h4
+          .list
+            .list_item
+              .list_link
+          .date .u-mb0
+          .short-desc .u-mb0
+  tags:
+    - cfgov-molecules
+*/
+
+@margin_half__em: unit(15px / @base-font-size-px, em);
+
+.m-related-metadata {
+    .list_link {
+        .u-link__colors(@pacific, @pacific, @pacific-50, @pacific, @navy);
+    }
+
+    .respond-to-range(@bp-sm-min, @bp-sm-max, {
+        border-left: @margin_half__em solid transparent;
```

Comment:
I could use `.grid_column(12)` but I would also need to use `.grid_nested-col-group()` when using the `half-width` modifier. This is the simplest yet ugliest way that I know.